Makefile

```diff
@@ -51,6 +51,17 @@ man:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+test-style: test-fmt test-vet test-lint
+
+test-fmt:
+	test -z "$$(gofmt -s -l . | grep -v ^vendor/ | tee /dev/stderr)"
```

Comment:
See more in the top-level conversation; I didn't notice the `validate` target.

Still, the difference between ~7 seconds for `sudo make validate` (and _much_ more for an initial uncached run) and ~1.5 seconds for a non-container run is pretty noticeable.

Or perhaps we should have IDE configuration running the checks locally during development, and `Makefile` targets for release compilation and end users? But then which IDE?

(I would _so_ love to have all of this figured out by some build system specialist instead of reinventing/cargo-culting for every project again.)